combobulate-ocaml.el

```diff
@@ -309,145 +309,35 @@
-                     :match-children t))
+              
 
         (:activation-nodes
         ((:nodes ("module_binding" "module_name") :has-ancestor ("functor")) )
         :selector (:choose
                     node
                     :match-children t))
 
-        (:activation-nodes
-            ((:nodes ("module_definition") :has-parent ("structure")) )
-            :selector (:choose
-                      node
-                      :match-children t))
-
-        (:activation-nodes
-          ((:nodes ("structure" "signature") :has-parent ("module-binding")))
-          :selector (:choose
-                     node
-                     :match-children t))
-
-        (:activation-nodes
-          ((:nodes ("structure" "signature")))
-          :selector (:choose
-                     node
-                     :match-children t))
-
         (:activation-nodes
           ((:nodes ("functor" )))
           :selector (:choose
                      node
                      :match-children (:discard-rules ("module_parameter" "struct"))))
 
         (:activation-nodes
-          ((:nodes ("function_type")
-                   :position any))
-          :selector
-          (:choose node
-                   :match-children t))
-
-         (:activation-nodes
-          ((:nodes ("match_expression" "function_type")))
-          :selector (:choose
-                     node
-                     :match-children t))
-
-      (:activation-nodes
-          ((:nodes ("module_definition" "module_binding" "module_name")) )
-          :selector (:choose
-                     node
-                     :match-children t))
-
-       (:activation-nodes
-          ((:nodes ((rule "polymorphic_variant_type"))
-                   :position at
-                   :has-parent ("polymorphic_variant_type")))
-          :selector (:choose parent :match-children t))
-
-      (:activation-nodes
-          ((:nodes ("object_expression" )))
+          ((:nodes ("class_definition" "class_type_definition" "class_binding" 
+                    "class_type_binding" "class_type_name" "class_body_type" "instance_variable_definition" "functor" "match_expression" "function_type" "module_definition" "module_binding" "module_name" "structure" "signature" "type_definition" "value_specification" "type_constructor_path" "type_binding" "let_binding" "type_constructor" "polymorphic_variant_type" "set_expression" "infix_expression" "application_expression" "expression_item" "let_expression" "value_definition" "match_case" "guard" "function_expression" "object_expression" "method_definition" "method_specification" "method_name" "inheritance_specification" "instantiated_class_type"
+                    "instance_variable_specification" "type_parameter_constraint" "record_declaration" "field_declaration" "variant_declaration" "constructor_declaration" "constructor_name" "attribute" "floating_attribute" "attribute_id" "attribute_payload" "field_name" "instance_variable_name"
```

Comment:
This could be simplified using `(rule "block")` to lookup the common parent node.  https://github.com/mickeynp/combobulate/wiki/Adding-a-New-Language-to-Combobulate#rules